package.json

```diff
@@ -1,11 +1,35 @@
 {
+  "name": "gql-gateway",
+  "description": "GraphQL Gateway that does magic on top of REST services that provides Swagger/OpenAPI specifications.",
+  "keywords": [
+    "graphql",
+    "gateway",
+    "swagger"
+  ],
+  "version": "1.0.8",
```

Comment:
Can you bump the version to 1.0.9 ?

Reply:
Thank for the review @segpacto. 
Version upgraded to 1.0.9.